ingestion/src/metadata/ingestion/source/messaging/kinesis/metadata.py

```diff
@@ -160,9 +172,11 @@ def _get_sample_data(self, topic_name, partitions) -> TopicSampleData:
                     "Records"
                 ]
 
-                data.extend(
-                    [b64decode(record["Data"]).decode("utf-8") for record in records]
-                )
+                for record in records:
+                    try:
+                        data.append(b64decode(record["Data"]).decode("utf-8"))
```

Comment:
for utf-8 we can use the constant `metadata.utils.constant.UTF_8`

Reply:
Updated the constant